test/edu-benefits/5490/config/benefitHistory.unit.spec.jsx

```diff
@@ -57,7 +57,7 @@ describe('Edu 5490 benefitHistory', () => {
 
     // Should expand to 16
     expect(Array.from(formDOM.querySelectorAll('input,select')).length)
-      .to.equal(16);
+      .to.equal(17);
```

Comment:
I think we should also have a test that checks the required states for those two fields change appropriately.